### PR TITLE
Translate to indonesian language

### PR DESCRIPTION
**Pull request by ppabcd**
Translate Install Ubuntu Desktop to Indonesia Language

### CODE REVIEW

**Comment by elopio** [comment]
Thanks for you work @ppabcd!
Instead of editing the english file, can you please move your translation to a directory named "id"? Take a look at how it's done for the french tutorial:
https://github.com/canonical-websites/tutorials.ubuntu.com/tree/master/tutorials/fr/setting-up-lxd-1604-fr

**Comment by ppabcd** [comment]
ok i will move my translation to that directory.

File: tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+published: 2017-07-06
+author: Canonical Web Team <webteam@canonical.com>
+translator: Reza Juliandri
+
+---
+
+# Memasang Ubuntu desktop
+
+## Pendahuluan
+Durasi: 0:05
```
Comment (caldav) [inline tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md]: Please change "Duration" back to english, this won't be display to users, but used internally by the tutorials engine, thanks!

File: tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+Ubuntu desktop mudah digunakan, mudah di pasang dan juga memuat apapun yang kamu butuhkan untuk organisasi, sekolah,
+rumah atau perusahaan. Serta lisensi terbuka, aman, mudah digunakan dan juga  gratis di unduh.
+
+![screenshot](https://assets.ubuntu.com/v1/8de0a986-ubuntu-desktop.png)
+
+Di tutorial ini, kami akan melakukan pemasangan Ubuntu desktop ke dalam komputer kamu, menggunakan drive DVD komputer ataupun menggunakan drive USB.
+
+
+## Persyaratan
+Durasi: 0:01
```
Comment (caldav) [inline tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md]: Please change "Duration" back to english

File: tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+* Memiliki akses ke DVD atau drive USB yang berisi versi ubuntu yang anda inginkan untuk dipasang.
+* Pastikan kamu memiliki cadangan data kamu baru-baru ini. Meskipun tidak mungkin
+  apapun akan salah, kamu tidak akan pernah bisa untuk siap.
+
+Lihat [Installation/System Requirements](https://help.ubuntu.com/community/Installation/SystemRequirements) untuk
+spesifik lebih lanjut mengenai persyaratan hardware. Kami juga memiliki [beberapa tutorial](/) yang menjelaskan bagaimana membuat Ubuntu DVD atau drive USB.
+
+
+## Boot dari DVD
+Durasi: 0:03
```
Comment (caldav) [inline tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md]: Please change "Duration" back to english

File: tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+
+Tunggu beberapa saat, setelah desktop termuat, kamu akan melihat jendela selamat datang. Dari sini kamu bisa memilih bahasa yang berada di list sebelah kiri dan memilih diantara menginstal Ubuntu secara langsung, atau mencoba desktop terlebih dahulu (Jika kamu menyukai apa yang kamu liat, kamu bisa memasang Ubuntu dari mode ini juga).
+
+
+![screenshot](https://assets.ubuntu.com/v1/dfc572f0-download-desktop-install-ubuntu-desktop_1.jpg)
+
+Jika kamu tidak mendapatkan menu ini, baca [panduan booting dari DVD](https://help.ubuntu.com/community/BootFromCD?_ga=2.102380610.2115462233.1496186978-1155966827.1485186360) untuk info lebih lanjut.
+
+## Boot dari Drive USB
+Durasi: 0:02
```
Comment (caldav) [inline tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md]: Please change "Duration" back to english

File: tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+
+Kebanyakan komputer akan boot dari USB secara automatis. Masukkan drive USB dan daya pada komputer kamu atau restart. Kamu akan melihat jendela selamat datang seperti kita lihat di sebelumnya di langkah 'Install dari DVD', menampilkan kamu untuk memilih bahasa dan menginstal Ubuntu secara langsung, atau mencoba desktop terlebih dahulu.
+
+Jika komputer kamu tidak automatis boot dari USB, coba tekan `F12` ketika komputer kamu memulai. Di banyak mesin ini akan mengizinkan kamu untuk memilih USB dari system-spesific boot menu.
+
+positif
+: F12 adalah kunci paling umum untuk menampilkan boot menu, tetapi gabungan Escape dan F2 bisa menjadi alternatif. Jika kamu tidak yakin, lihat pesan singkat ketika memulai sistem - ini akan memberikan informasi kepada kamu mengenai kunci yang harus di tekan untuk menuju boot menu.
+
+## Persiapan memasang Ubuntu
+Durasi: 0:01
```
Comment (caldav) [inline tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md]: Please change "Duration" back to english

File: tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+
+- Kami memberitahu mengaktifkan kedua `Pembaharuan unduhan` dan `Pemasangan perangkat lunak pihak-ketiga`.
+- Tetap terhubung ke internet jadi kamu bisa mendapatkan pembaharuan terbaru ketika  memasang Ubuntu.
+- Jika kamu tidak terkoneksi internet, kamu akan ditanyakan untuk memilih wireless network, jika tersedia. Kami memberitahu kamu untuk terkoneksi ketika melakukan pemasangan jadi kami bisa memastikan mesin up to date.
+
+
+![screenshot](https://assets.ubuntu.com/v1/3bbb0e35-download-desktop-install-ubuntu-desktop_2.jpg)
+
+## Alokasi ruang drive
+Durasi 0:01
```
Comment (caldav) [inline tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md]: Please change "Duration" back to english

File: tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md [hunk 7]
```diff
@@ -0,0 +1,159 @@
...
+Gunakan kotak centang untuk memilih seperti Memasang Ubuntu di samping operasi sistem lain, hapus kotak centang sistem operasi yang kamu miliki dan ganti dengan Ubuntu, atau — jika kamu adalah pengguna advanced  — pilih opsi  ’**Something else**’.
+
+![screenshot](https://assets.ubuntu.com/v1/e07b98b8-ubuntu-installation-type.png)
+
+positif
+: Ubuntu membutuhkan 4.5 GB ruang untuk minimal pemasangan. Tetapi  kamu akan membutuhkan ruang untuk data dan tambahan aplikasi yang kamu pasang.
+positive
+
+## Memulai Installasi
+Durasi: 0:01
```
Comment (caldav) [inline tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md]: Please change "Duration" back to english

File: tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md [hunk 8]
```diff
@@ -0,0 +1,159 @@
...
+
+Untuk memanggil Ubuntu tata letak keyboard kamu, klik di pengaturan bahasa kamu butuh mengikuti tata letak spesifik keyboard kamu. Jika kamu tidak yakin, klik tombol '**Deteksi Tata Letak Keyboard**' untuk bantuan.
+
+Tata letak keyboard bisa selalu berubah, dan tata letak tambahan ditambahkan, dari desktop setelah instalasi.
+
+![screenshot](https://assets.ubuntu.com/v1/310b5196-download-desktop-install-ubuntu-desktop_7.jpg)
+
+## Informasi Login
+Durasi: 0:01
```
Comment (caldav) [inline tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md]: Please change "Duration" back to english

File: tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md [hunk 9]
```diff
@@ -0,0 +1,159 @@
...
+
+Jika kamu tidak yakin dengan zona waktu kamu, tulis nama kota lokal atau kota atau gunakan map untuk memilih lokasi.
+
+![screenshot](https://assets.ubuntu.com/v1/a942aa3d-download-desktop-install-ubuntu-desktop_6.jpg)
+
+positif
+: Jika kamu memiliki masalah mengkoneksikan ke Intenet, gunakan menu di tangan-atas-kanan sudut untuk memilih jaringan.
+
+## Pilih preferensi tata letak keyboard
+Durasi: 0:01
```
Comment (caldav) [inline tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md]: Please change "Duration" back to english

File: tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md [hunk 10]
```diff
@@ -0,0 +1,159 @@
...
+
+Kamu bisa memilih automatis login dan enskripsi folder rumah. Jika mesin kamu portable, kami merekomendasikan tetap automatis login dimatikan atau menyalakan enskripsi. Ini akan menghentikan orang lain mengakses file personal kamu jika mesin hilang atau dicuri.
+
+
+![screenshot](https://assets.ubuntu.com/v1/69d32ea7-download-desktop-install-ubuntu-desktop_8.jpg)
+
+: If enable home folder encryption and you forget your password, you won't be able to retrieve any personal data stored in your home folder.
+
+## Installasi di latar belakang
+Durasi: 0:20
```
Comment (caldav) [inline tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md]: Please change "Duration" back to english

File: tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md [hunk 11]
```diff
@@ -0,0 +1,159 @@
...
+
+Pemasang akan selesai di latar belakang ketika jendela pemasangan mengajari kamu sedikit tentang bagaimana mengagumkannya Ubuntu. Bergantung kecepatan mesin kamu dan koneksi jaringan, ini hanya akan memakan waktu beberapa menit.
+
+![screenshot](https://assets.ubuntu.com/v1/4e133117-download-desktop-install-ubuntu-desktop_9.jpg)
+
+## Installasi selesai
+Durasi: 0:01
```
Comment (caldav) [inline tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md]: Please change "Duration" back to english

**Comment by caldav** [comment]
Hi @ppabcd and thanks! I took the liberty to add an extra tag to your tutorial metadata. See the rationale for it in issue #509

**Comment by KazemitoHaruhi** [comment]
seems good.

File: tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md [hunk 12]
```diff
@@ -0,0 +1,159 @@
...
+Duration: 0:20
+
+Pemasang akan selesai di latar belakang ketika jendela pemasangan mengajari kamu sedikit tentang bagaimana mengagumkannya Ubuntu. Bergantung kecepatan mesin kamu dan koneksi jaringan, ini hanya akan memakan waktu beberapa menit.
+
+![screenshot](https://assets.ubuntu.com/v1/4e133117-download-desktop-install-ubuntu-desktop_9.jpg)
+
+## Installasi selesai
+Duration: 0:01
+
+Setelah semuanya terpasang dan terkonfigurasi, jendela kecil akan menampilkan pertanyaan untuk melakukan restart mesin. Klik `Restart Now` dan hapus diantara DVD atau drive USB.
```
Comment (atriwidada) [inline tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md]: remove either the DVD or USB flash drive when promoted -> keluarkan DVD atau lepaskan flash disk USB ketika diminta
(promoted di sini kelihatannya salah ketik, dugaan saya 'prompted')

File: tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md [hunk 13]
```diff
@@ -0,0 +1,159 @@
...
+
+
+![screenshot](https://assets.ubuntu.com/v1/f4e2a592-download-desktop-install-ubuntu-desktop_10.jpg)
+Selamat ! Kamu berhasil memasang sistem  opeasi Linux paling populer di dunia!
```
Comment (atriwidada) [inline tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md]: Antara 'Selamat' dan tanda seru tidak boleh ada spasi
opeasi -> operasi
(spasi)(spasi)opeasi -> (spasi)operasi

File: tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md [hunk 14]
```diff
@@ -0,0 +1,159 @@
...
+Congratulations! You have successfully installed the world's most popular Linux operating system!
```
Comment (atriwidada) [inline tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md]: baris ini mestinya dihapus?

File: tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md [hunk 15]
```diff
@@ -0,0 +1,159 @@
...
+
+Sekarang adalah waktu untuk memulai menikmati Ubuntu!
+
+### Mencari Bantuan
+
+Jika kamu mengalami masalah, bantuan selalu ada di tangan.
+
+* [Ask Ubuntu](https://askubuntu.com/)
+* [Ubuntu Forums](https://ubuntuforums.org/)
```
Comment (atriwidada) [inline tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md]: Ubuntu Forums -> Forum Ubuntu

File: tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md [hunk 16]
```diff
@@ -0,0 +1,159 @@
...
+* [IRC-based support](https://wiki.ubuntu.com/IRC/ChannelList)
```
Comment (atriwidada) [inline tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md]: IRC-based support -> Dukungan berbasis IRC

Comment (atriwidada) [reply tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md]: Please disregard my review about 'Duration -> Durasi', I haven't read caldav's comment.

File: tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md [hunk 17]
```diff
@@ -0,0 +1,159 @@
...
+published: 2017-07-06
+author: Canonical Web Team <webteam@canonical.com>
+translator: Reza Juliandri
+
+---
+
+# Memasang Ubuntu desktop
+
+## Pendahuluan
+Duration: 0:05
```
[the remark this answers is not in the document]
Comment (caldav) [reply tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md]: Duration should stay in english, it's used by the website and not exposed to readers, thanks!

File: tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md [hunk 18]
```diff
@@ -0,0 +1,159 @@
...
+Ubuntu desktop sangat mudah digunakan, mudah di pasang dan juga memuat apapun yang kamu butuhkan untuk organisasi, sekolah,
+rumah atau perusahaan. Serta lisensi terbuka, aman, mudah digunakan dan juga  gratis diunduh.
+
+![screenshot](https://assets.ubuntu.com/v1/8de0a986-ubuntu-desktop.png)
+
+Di tutorial ini, kami akan melakukan pemasangan Ubuntu desktop ke dalam komputer kamu, menggunakan drive DVD komputer maupun menggunakan drive USB.
+
+
+## Persyaratan
+Duration: 0:01
```
[the remark this answers is not in the document]
Comment (caldav) [reply tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md]: Same here, please keep Duration

File: tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md [hunk 19]
```diff
@@ -0,0 +1,159 @@
...
+* Memiliki akses ke DVD atau drive USB yang berisi versi ubuntu yang anda inginkan untuk dipasang.
+* Pastikan kamu memiliki cadangan data kamu baru-baru ini. Meskipun tidak mungkin
+  apapun akan salah, kamu tidak akan pernah bisa untuk siap.
+
+Lihat [Installation/System Requirements](https://help.ubuntu.com/community/Installation/SystemRequirements) untuk
+spesifik lebih lanjut mengenai persyaratan hardware. Kami juga memiliki [beberapa tutorial](/) yang menjelaskan bagaimana membuat Ubuntu DVD atau drive USB.
+
+
+## Boot dari DVD
+Duration: 0:03
```
[the remark this answers is not in the document]
Comment (caldav) [reply tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md]: Same here, please keep Duration

File: tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md [hunk 20]
```diff
@@ -0,0 +1,159 @@
...
+
+Tunggu beberapa saat, setelah desktop termuat, kamu akan melihat jendela selamat datang. Dari sini kamu bisa memilih bahasa yang berada di list sebelah kiri dan memilih diantara menginstal Ubuntu secara langsung, atau mencoba desktop terlebih dahulu (Jika kamu menyukai apa yang kamu liat, kamu bisa memasang Ubuntu dari mode ini juga).
+
+
+![screenshot](https://assets.ubuntu.com/v1/dfc572f0-download-desktop-install-ubuntu-desktop_1.jpg)
+
+Jika kamu tidak mendapatkan menu ini, baca [panduan booting dari DVD](https://help.ubuntu.com/community/BootFromCD?_ga=2.102380610.2115462233.1496186978-1155966827.1485186360) untuk info lebih lanjut.
+
+## Boot dari Drive USB
+Duration: 0:02
```
[the remark this answers is not in the document]
Comment (caldav) [reply tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md]: Same here, please keep Duration

File: tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md [hunk 21]
```diff
@@ -0,0 +1,159 @@
...
+
+Kebanyakan komputer akan boot dari USB secara automatis. Masukkan drive USB dan daya pada komputer kamu atau restart. Kamu akan melihat jendela selamat datang seperti kita lihat di sebelumnya di langkah 'Install dari DVD', menampilkan kamu untuk memilih bahasa dan menginstal Ubuntu secara langsung, atau mencoba desktop terlebih dahulu.
+
+Jika komputer kamu tidak automatis boot dari USB, coba tekan `F12` ketika komputer kamu memulai. Di banyak mesin ini akan mengizinkan kamu untuk memilih USB dari system-spesific boot menu.
+
+positif
+: F12 adalah kunci paling umum untuk menampilkan boot menu, tetapi gabungan Escape dan F2 bisa menjadi alternatif. Jika kamu tidak yakin, lihat pesan singkat ketika memulai sistem - ini akan memberikan informasi kepada kamu mengenai kunci yang harus di tekan untuk menuju boot menu.
+
+## Persiapan memasang Ubuntu
+Duration: 0:01
```
[the remark this answers is not in the document]
Comment (caldav) [reply tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md]: Same here, please keep Duration

File: tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md [hunk 22]
```diff
@@ -0,0 +1,159 @@
...
+
+- Kami memberitahu mengaktifkan kedua `Pembaharuan unduhan` dan `Pemasangan perangkat lunak pihak-ketiga`.
+- Tetap terhubung ke internet jadi kamu bisa mendapatkan pembaharuan terbaru ketika  memasang Ubuntu.
+- Jika kamu tidak terkoneksi internet, kamu akan ditanyakan untuk memilih wireless network, jika tersedia. Kami memberitahu kamu untuk terkoneksi ketika melakukan pemasangan jadi kami bisa memastikan mesin up to date.
+
+
+![screenshot](https://assets.ubuntu.com/v1/3bbb0e35-download-desktop-install-ubuntu-desktop_2.jpg)
+
+## Alokasi ruang drive
+Duration: 0:01
```
[the remark this answers is not in the document]
Comment (caldav) [reply tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md]: Same here, please keep Duration

File: tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md [hunk 23]
```diff
@@ -0,0 +1,159 @@
...
+Gunakan kotak centang untuk memilih seperti Memasang Ubuntu di samping operasi sistem lain, hapus kotak centang sistem operasi yang kamu miliki dan ganti dengan Ubuntu, atau — jika kamu adalah pengguna advanced  — pilih opsi  ’**Something else**’.
+
+![screenshot](https://assets.ubuntu.com/v1/e07b98b8-ubuntu-installation-type.png)
+
+positif
+: Ubuntu membutuhkan 4.5 GB ruang untuk minimal pemasangan. Tetapi  kamu akan membutuhkan ruang untuk data dan tambahan aplikasi yang kamu pasang.
+positive
+
+## Memulai Installasi
+Duration: 0:01
```
[the remark this answers is not in the document]
Comment (caldav) [reply tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md]: Same here, please keep Duration

File: tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md [hunk 24]
```diff
@@ -0,0 +1,159 @@
...
+
+Setelah konfigurasi penyimpanan. klik tombol 'Install sekarang'. Jendela kecil akan muncul dengan petinjauan opsi penyimpanan kamu pilih, dengan memungkinkan pergi kembali jika ada detail yang salah.
+
+Klik `Lanjutkan` untuk membenarkan perubahan di tempat dan memulai proses installasi.
+
+![screenshot](https://assets.ubuntu.com/v1/a76b68ce-ubuntu-verify-install.png)
+
+## Pilih lokasi kamu
+Duration: 0:01
```
[the remark this answers is not in the document]
Comment (caldav) [reply tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md]: Same here, please keep Duration

File: tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md [hunk 25]
```diff
@@ -0,0 +1,159 @@
...
+
+Jika kamu tidak yakin dengan zona waktu kamu, tulis nama kota lokal atau kota atau gunakan map untuk memilih lokasi.
+
+![screenshot](https://assets.ubuntu.com/v1/a942aa3d-download-desktop-install-ubuntu-desktop_6.jpg)
+
+positif
+: Jika kamu memiliki masalah mengkoneksikan ke Intenet, gunakan menu di tangan-atas-kanan sudut untuk memilih jaringan.
+
+## Pilih preferensi tata letak keyboard
+Duration: 0:01
```
[the remark this answers is not in the document]
Comment (caldav) [reply tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md]: Same here, please keep Duration

File: tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md [hunk 26]
```diff
@@ -0,0 +1,159 @@
...
+
+Untuk memanggil Ubuntu tata letak keyboard kamu, klik di pengaturan bahasa kamu butuh mengikuti tata letak spesifik keyboard kamu. Jika kamu tidak yakin, klik tombol '**Deteksi Tata Letak Keyboard**' untuk bantuan.
+
+Tata letak keyboard bisa selalu berubah, dan tata letak tambahan ditambahkan, dari desktop setelah instalasi.
+
+![screenshot](https://assets.ubuntu.com/v1/310b5196-download-desktop-install-ubuntu-desktop_7.jpg)
+
+## Informasi Login
+Duration: 0:01
```
[the remark this answers is not in the document]
Comment (caldav) [reply tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md]: Same here, please keep Duration

File: tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md [hunk 27]
```diff
@@ -0,0 +1,159 @@
...
+
+## Installasi di latar belakang
+Duration: 0:20
+
+Pemasang akan selesai di latar belakang ketika jendela pemasangan mengajari kamu sedikit tentang bagaimana mengagumkannya Ubuntu. Bergantung kecepatan mesin kamu dan koneksi jaringan, ini hanya akan memakan waktu beberapa menit.
+
+![screenshot](https://assets.ubuntu.com/v1/4e133117-download-desktop-install-ubuntu-desktop_9.jpg)
+
+## Installasi selesai
+Duration: 0:01
```
[the remark this answers is not in the document]
Comment (caldav) [reply tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md]: Same here, please keep Duration

File: tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md [hunk 28]
```diff
@@ -0,0 +1,159 @@
...
+
+Kamu bisa memilih automatis login dan enskripsi folder rumah. Jika mesin kamu portable, kami merekomendasikan tetap automatis login dimatikan atau menyalakan enskripsi. Ini akan menghentikan orang lain mengakses file personal kamu jika mesin hilang atau dicuri.
+
+
+![screenshot](https://assets.ubuntu.com/v1/69d32ea7-download-desktop-install-ubuntu-desktop_8.jpg)
+
+: If enable home folder encryption and you forget your password, you won't be able to retrieve any personal data stored in your home folder.
+
+## Installasi di latar belakang
+Duration: 0:20
```
[the remark this answers is not in the document]
Comment (caldav) [reply tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md]: Same here, please keep Duration

**Comment by ppabcd** [comment]
almost done...

File: tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md [hunk 29]
```diff
@@ -0,0 +1,155 @@
...
+- Tetaplah terhubung ke internet jadi kamu bisa mendapatkan pembaharuan terakhir ketika memasang Ubuntu.
+- Jika kamu tidak terkoneksi ke internet, kamu akan ditanyakan untuk memilih jaringan nirkabel, jika tersedia. Kami sarankan agar terkoneksi ketika melakukan pemasangan jadi kami bisa memastikan mesin mutakhir.
+
+
+![screenshot](https://assets.ubuntu.com/v1/3bbb0e35-download-desktop-install-ubuntu-desktop_2.jpg)
+
+## Alokasi ruang drive
+Duration: 0:01
+
+Gunakan kotak centang untuk memilih apakah kamu ingin Memasang Ubuntu berdampingan dengan sistem operasi lain, menghapus sistem operasi yang Anda dan menggantikannya dengan Ubuntu, atau — jika kamu adalah pengguna tingkat lanjut  — pilih opsi  ’**Something else**’.
```
Comment (atriwidada) [inline tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md]: sori, review saya sebelumnya ternyata ada typo
delete your existing operating system and replace it with Ubuntu -> menghapus sistem operasi yang ada dan menggantikannya dengan Ubuntu

File: tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md [hunk 30]
```diff
@@ -0,0 +1,155 @@
...
+Duration: 0:01
+
+Jika kamu terkoneksi dengan internet, lokasi kamu akan terdeteksi otomatis. Cek lokasi kamu jika benar dan klik 'Meneruskan' untuk memproses.
+
+Jika kamu tidak yakin dengan zona waktu kamu, tulis nama kota lokal atau kota atau gunakan peta untuk memilih lokasi.
+
+![screenshot](https://assets.ubuntu.com/v1/a942aa3d-download-desktop-install-ubuntu-desktop_6.jpg)
+
+positif
+: Jika kamu memiliki masalah mengkoneksikan ke Intenet, gunakan menu di pojok kanan atas sudut untuk memilih jaringan.
```
Comment (atriwidada) [inline tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md]: pojok kanan atas sudut -> pojok kanan atas

Comment (atriwidada) [reply tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md]: Intenet -> Internet

File: tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md [hunk 31]
```diff
@@ -0,0 +1,155 @@
...
+![screenshot](https://assets.ubuntu.com/v1/310b5196-download-desktop-install-ubuntu-desktop_7.jpg)
+
+## Informasi Login
+Duration: 0:01
+
+Masukkan nama kamu dan pemasang akan otomatis mensugestikan nama komputer dan username. Ini bisa berubah dengan mudah jika kamu lebih suka. Nama komputer adalah bagaimana komputer akan muncul di jaringan, Username kamu akan menjadi login kamu dan nama akun.
+
+Selanjutnya, masukkan password yang kuat. Pemasang akan memberitahu jika itu sangat lemah.
+
+Kamu bisa memilih otomatis login dan enkripsi folder rumah. Jika mesin kamu portable, kami merekomendasikan tetap login otomatis dimatikan atau menyalakan enskripsi. Ini akan mencegah orang lain mengakses file personal kamu jika mesin hilang atau dicuri.
```
Comment (atriwidada) [inline tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md]: we recommend keeping automatic login disabled -> kami merekomendasikan agar login otomatis dimatikan

File: tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md [hunk 32]
```diff
@@ -0,0 +1,155 @@
...
+Duration: 0:20
+
+Pemasang akan selesai di latar belakang ketika jendela pemasangan mengajari kamu sedikit tentang bagaimana mengagumkannya Ubuntu. Bergantung kecepatan mesin kamu dan koneksi jaringan, ini hanya akan memakan waktu beberapa menit.
+
+![screenshot](https://assets.ubuntu.com/v1/4e133117-download-desktop-install-ubuntu-desktop_9.jpg)
+
+## Instalasi selesai
+Duration: 0:01
+
+Setelah semuanya terpasang dan terkonfigurasi, jendela kecil akan menampilkan pertanyaan untuk melakukan restart mesin. Klik `Restart Now` dan hkeluarkan DVD atau lepaskan flash disk USB ketika diminta.
```
Comment (atriwidada) [inline tutorials/id/install-ubuntu-desktop/install-ubuntu-desktop-1604-tutorial-id.md]: hkeluarkan -> keluarkan

**Comment by caldav** [comment]
@ppabcd, thanks for your contribution and @atriwidada for the review! +1 from me

**Comment by ppabcd** [comment]
Your welcome :)

**Comment by elopio** [comment]
Thank you @ppabcd @atriwidada and @caldav <3

**Comment by ppabcd** [comment]
Your welcome :)

**Pull request merged**